packages/shared/src/components/PromotionalBanner.tsx

```diff
@@ -21,15 +21,15 @@ const classNamesByTheme: Record<BannerTheme, string[]> = {
     'bg-theme-bg-reverse',
     'text-theme-label-invert',
   ],
-  [Theme.Avocado]: ['bg-theme-color-avocado', 'text-pepper-90'],
+  [Theme.Avocado]: ['bg-theme-color-avocado', 'text-raw-pepper-90'],
```

Comment:
What is the idea with `-raw`, do they call it in figma like that?

Reply:
No they are the "raw" colors which we technically should never use directly.
It's just renamed to make it easier to disable/filter out.

Reply:
Yep mostly made it impossible to clean the story basically.
I think by having it under "raw" we can also more easily lint that those should not be allowed directly in classnames. :) 
Let me know if that works, or you see issues with it?

Reply:
All good!